cmake/HPX_AddConfigTest.cmake

```diff
@@ -503,15 +407,6 @@ function(hpx_check_for_cxx17_shared_ptr_array)
   )
 endfunction()
 
-# ##############################################################################
-function(hpx_check_for_cxx17_std_nontype_template_parameter_auto)
```

Comment:
Same here. I know that MSVC is still struggling with this in certain scenarios. This might not be an issue for us, however.